docs/DeveloperGuide.md

```diff
@@ -591,42 +589,116 @@ testers are expected to do more *exploratory* testing.
 
 ### Launch and shutdown
 
-1. Initial launch
+1. Initial launch.
 
-   1. Download the jar file and copy into an empty folder
+    1. Download the jar file and copy it into an empty folder.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+    2. Double-click the jar file. <br>
+       Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences.
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+    2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Adding a batchmate
+
+1. Adding a batchmate to the data.
+
+    1. Test case: `add n/John t/john`.<br>
+       Expected: A new batchmate with the name John and telegram handle john is added at the bottom of the students pane in MassLinkers.
```

Comment:
Not sure if we should quote John and john as `John` and `john` here, i.e. A new batchmate with the name `John` and telegram handle `john` is added at the bottom...
This style follows the User Guide style.

Same for those below.